src/libLookup/Lookup.cpp

```diff
@@ -1655,7 +1655,8 @@ bool Lookup::ProcessGetCosigsRewardsFromSeed(
   uint32_t retryCount = 5;
   while (retryCount-- > 0) {
     if (!BlockStorage::GetBlockStorage().GetTxBlock(blockNum, txblkPtr)) {
-      LOG_GENERAL(WARNING, "Failed to fetch tx block, retry... " << blockNum);
+      LOG_GENERAL(WARNING,
```

Comment:
Do we need some verification for this message, eg Checking pubkey of lookup??

Reply:
This message is processed by lookup itself. This message is verified for sender's pubkey to be part  ds committee.